backend/cmd/migrate/migrations/000004_create_topology_cache_table.up.sql

```diff
@@ -0,0 +1,32 @@
+CREATE OR REPLACE FUNCTION set_timestamp() RETURNS TRIGGER LANGUAGE plpgsql AS '
+BEGIN
+  NEW.updated_at = NOW();
+  RETURN NEW;
+END;
+';
```

Comment:
can we avoid triggers and do this type of thing server-side?

Reply:
Sure